common/lib/xmodule/xmodule/tests/test_course_metadata_utils.py

```diff
@@ -159,6 +160,10 @@ def mock_strftime_localized(date_time, format_string):
                     advertised_start_unparsable.title()
                 ),
                 TestScenario(
+                    (test_datetime, advertised_start_bad_date, 'DATE_TIME', ugettext, mock_strftime_localized),
```

Comment:
Yes it does throw a `ValueError` on `advertised_start_bad_date`, and that's indirectly tested by this `TestScenario`. I'll add a more explicit test though.